### PR TITLE
Refactor to DependencyIjection to support easier Varmegyle or Eirele

### PR DESCRIPTION
**Pull request by szalabala**

### CODE REVIEW

File: src/components/GameRoundPot/GameRoundPot.tsx [hunk 1]
```diff
@@ -108,6 +107,8 @@ function GameRoundPot({
     console.log("Guess button clicked.");
   };
 
+  console.log(`typeof getPot... ${typeof dataBank.getPotMapSvgUrl}`);
```
Comment (ppesti) [inline src/components/GameRoundPot/GameRoundPot.tsx:110]: might not be needed

File: src/components/GameRoundNeighbors/GameRoundNeighbors.tsx [hunk 2]
```diff
@@ -195,7 +187,8 @@ function GameRoundNeighbors({
           <div className="grid grid-cols-6 gap-1 text-center py-0.5">
             <div className="my-div-1">
               <span className="opacity-70">
-                {t("guessNoun")} {guesses.length + 1} / {maxAttempts}
+                {dataBank.tLang("guessNoun")} {guesses.length + 1} /{" "}
```
Comment (ppesti) [inline src/components/GameRoundNeighbors/GameRoundNeighbors.tsx:190]: prettier turned into uglier, perhaps undo these changes (it's on multiple lines)

**Review by ppesti — APPROVED** [review]

File: src/components/GameRoundNeighbors/GameRoundNeighbors.tsx (hunk 2 above)
Comment (szalabala) [reply src/components/GameRoundNeighbors/GameRoundNeighbors.tsx]: done
